pkg/kube/inject/webhook.go

```diff
@@ -250,6 +250,7 @@ func (wh *Webhook) Run(stop <-chan struct{}) {
 	for {
 		select {
 		case <-timerC:
+			log.Info("Injector watch update")
```

Comment:
this will be very frequent. kill it? or make it debug?

Reply:
I think its only when mesh config or injection template changes which is not that common? before its logging like 5 times per change, now we log once and after the debounce. I can make it debug if you think its too much though, I don't care too much

Reply:
Actually it happens pretty often.. not sure why, since the template isn't changing. I made it debug